packages/expo-notifications/ios/EXNotifications/Notifications/Categories/CategoriesModule.swift

```diff
@@ -47,9 +45,9 @@ open class CategoriesModule: Module {
       return oldCategory.identifier == identifier
     }
     if didDelete {
-      let newCategories = Set(oldCategories.filter { oldCategory in
```

Comment:
`oldCategories` is already a `Set`, so it's not necessary to create another `Set` out of it